Library/Formula/fpc.rb

```diff
@@ -25,11 +25,15 @@ def install
     system "make", "install", "PP=#{fpc_compiler}", "PREFIX=#{prefix}"
 
     bin.install_symlink lib/"#{name}/#{version}/ppcx64"
+
+    # generate a default fpc.cfg to facilitate search paths
+    system "./utils/fpcmkcfg/fpcmkcfg", "-p", "-d", "basepath=#{lib}/fpc/" + `./compiler/ppc1 -iW.chomp`, "-o", "#{prefix}/etc/fpc.cfg"
```

Comment:
```ruby
`./compiler/ppc1 -iW.chomp`
```

should be

```
`./compiler/ppc1 -iW`.chomp
```

Also, please pull out the `ppc1` into a separate variable and use `#{variable}` instead of `+`. Thanks!